radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/epoch/EpochManager.java

```diff
@@ -183,7 +183,8 @@ public EpochManager(
     this.syncErrorResponseProcessors =
         isValidator ? Set.of(initialBFTSync.errorResponseProcessor()) : Set.of();
     this.bftUpdateProcessors =
-        isValidator
+        isValidator // FIXME: initialBFTSync::processBFTUpdate does nothing, do we actually need it
```

Comment:
I'm concerned about increasing the number of untracked comments like this in the code base. Can we do one of:
* Investigate this now so we don't need to leave a comment, if you feel it fits with the current work you're doing
* Raise a ticket for this and mention the ticket number in the code comment

What are your thoughts?

Reply:
It requires somewhat deeper look before fix. This is out of scope of the current task.
I believe there is a dedicated ticket to transform all TODO's into tickets.

Reply:
I'm happy if we raise another ticket. But the best time to create that ticket is now whilst you have the context, to help whoever picks it up know where to start. Would you be happy to raise it?

Reply:
Actually, "FIXME" appeared exactly because I had no full context. Anyway, I've decided to look deeper and resolved the issue.